### ISSUE TITLE
Missing LIFX models in products.json

### ISSUE THREAD

**Issue by unreality** [node-lifx/lifx-lan-client#40]
Hi,

The following models are missing from products.json, can we get them added?

```javascript
      {
        "pid": 92,
        "name": "LIFX Color 1000lm",
        "features": {
          "color": true,
          "infrared": false,
          "multizone": false,
          "temperature_range": [
            2500,
            9000
          ],
          "chain": false
        }
      },
      {
        "pid": 97,
        "name": "LIFX Color A19 1200lm",
        "features": {
          "color": true,
          "infrared": false,
          "multizone": false,
          "temperature_range": [
            2500,
            9000
          ],
          "chain": false
        }
      }
```

**Comment by ristomatti** [comment]
Thank you for informing. Hopefully I'll have a moment to do it later today, now on mobile. Of course a pull request would also be welcome.

**Comment by ristomatti** [comment]
I've updated the file but I merged in a few dependency PR's so I'll still need to verify everything works OK before I publish this.

**Comment by ristomatti** [comment]
Published v1.1.0. Sorry it took so long!
